### DISCUSSION TITLE
Download links with Custom File Name

### DISCUSSION THREAD

**Discussion by TeamQuto** [anasty17/mirror-leech-telegram-bot#915] [Q&A]
![IMG_20220815_222825](https://user-images.githubusercontent.com/103844261/184681105-ea05335d-f283-4129-8444-c9b2675c4adf.jpg)

Other Leech Repo
![IMG_20220815_224145](https://user-images.githubusercontent.com/103844261/184682759-374e09f1-94ff-4eb1-96bc-db0a5dc06a47.jpg)

**Comment by anasty17** [comment]
Send link not related to rename

**Comment by anasty17** [comment]
Next time send logs
